src/utils/common.ts

```diff
@@ -59,6 +63,27 @@ export function logDev(message: unknown, ...optionalParams: unknown[]) {
   }
 }
 
+export const getCustomScreen = <T>(screenDict: ScreenMap<T>, item: PlaylistItem | Playlist | undefined) => {
```

Comment:
Remove 'custom'.  This is a way to customize, yes, but this function is likely getting core screens.  The customer customizing case is likely an exception to the rule.

Reply:
Yes agreed, when we differentiate pages and screens there is no need for the custom keyword at all